packages/sit-onyx/src/components/OnyxTable/OnyxTable.stories.ts

```diff
@@ -16,6 +17,7 @@ const meta: Meta<typeof OnyxTable> = {
         control: { disable: true },
       },
     },
+    decorators: [createTruncationDecorator("24rem")],
```

Comment:
can this example be excluded from the decorator?
![image](https://github.com/SchwarzIT/onyx/assets/151019977/76210c7f-9bec-46f1-95c6-89e8236e67c1)

we have 3 scrolling examples, this one should only have a scroll bar on the left.
the other on the bottom, the third both.
currently the first one also has both because of the 24rem

Reply:
fixed